cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.common.physical_server_views import PhysicalServersView
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="function")
+def physical_servers(appliance, provider, setup_provider):
+    # Get and return all the physical servers
+    return appliance.collections.physical_servers.all(provider)
+
+
+@pytest.fixture(scope="function")
+def physical_servers_collection(appliance):
+    # Get and return collection of the physical servers
+    return appliance.collections.physical_servers
+
+
+# Configuration Button
+def test_refresh_relationships(physical_servers_collection, physical_servers, provider):
+    view = provider.create_view(PhysicalServersView)
```

Comment:
**Required**

This should start with navigation, not just setting a view - there is no guarantee/enforcement of test case execution order, and the browser could be anywhere in the UI. The `last_refresh_date` call may navigate, but its risky to assume so, when it could be swapped out for a REST based method that never navigates.

Reply:
done